rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -66,11 +68,10 @@ rmw_init()
 {
   DDS::DomainParticipantFactory_var dp_factory = \
     DDS::DomainParticipantFactory::get_instance();
-  if (!dp_factory)
-  {
+  if (!dp_factory) {
```

Comment:
I would also prefer the third option. I finally found the corresponding uncrustify option: `nl_multi_line_cond`. I have updated the PRs where necessary.